.github/workflows/docker-build-ui-harbor.yml

```diff
@@ -0,0 +1,22 @@
+name: Build CAP UI image
+
+on:
+  push:
+    branches: next
+    paths:
+      - "ui/**"
+      - ".github/workflows/ui-test.yml"
+
+jobs:
+  build_dev:
+    name: Build ui image
+    runs-on: ubuntu-latest
+    steps:
+      - name: Build and Export
+        uses: cern-sis/gh-workflows/.github/actions/docker-build@v4.0.0
+        with:
+          file: ./docker/nginx/Dockerfile
+          image: cern-sis/cap-ui-dev
```

Comment:
Why not just `cern-sis/cap-ui` Our goal is to use this image in the whole pipeline from dev, qa and prod.